### PR TITLE
check_file_status->check_input_files

### PR DESCRIPTION
**Pull request by proteasome**

### CODE REVIEW

**Comment by proteasome** [comment]
please retest

**Comment by mr-c** [comment]
Jenkins, ok to test

**Comment by proteasome** [comment]
- [ ] Is it mergeable?
- [ ] Did it pass the tests?
- [ ] If it introduces new functionality in scripts/ is it tested?
  Check for code coverage with `make clean diff-cover`
- [ ] Is it well formatted? Look at `make pep8`, `make diff_pylint_report`,
  `make cppcheck`, and `make doc` output. Use `make format` and manual
  fixing as needed.
- [ ] Did it change the command-line interface? Only additions are allowed
  without a major version increment. Changing file formats also requires a
  major version number increment.
- [ ] Is it documented in the ChangeLog?
  http://en.wikipedia.org/wiki/Changelog#Format
- [ ] Was a spellchecker run on the source code and documentation after
  changes were made?

**Comment by proteasome** [comment]
ready for review!

**Comment by proteasome** [comment]
- [ ] Is it mergeable?
- [ ] Did it pass the tests?
- [ ] If it introduces new functionality in scripts/ is it tested?
  Check for code coverage with `make clean diff-cover`
- [ ] Is it well formatted? Look at `make pep8`, `make diff_pylint_report`,
  `make cppcheck`, and `make doc` output. Use `make format` and manual
  fixing as needed.
- [ ] Did it change the command-line interface? Only additions are allowed
  without a major version increment. Changing file formats also requires a
  major version number increment.
- [ ] Is it documented in the ChangeLog?
  http://en.wikipedia.org/wiki/Changelog#Format
- [ ] Was a spellchecker run on the source code and documentation after
  changes were made?
- [ ] Is the Copyright year up to date?

**Comment by proteasome** [comment]
ready for review!

**Comment by mr-c** [comment]
Looks good! I'm not seeing a ChangeLog entry, though.

**Comment by proteasome** [comment]
ChangeLog entry made
review for review!

**Comment by proteasome** [comment]
@mr-c ready to review!

File: ChangeLog [hunk 1]
```diff
@@ -1,3 +1,15 @@
+2015-04-15  en zyme  <en_zyme@outlook.com>
+   * khmer/khmer/kfile.py: check_file_status() -> check_input_files()
```
Comment (mr-c) [inline ChangeLog]: Looking great! I need a blank line above this one.

**Comment by proteasome** [comment]
@mr-c ready for review!

**Comment by mr-c** [comment]
@proteasome Congratulations on your first commit to the khmer project! Your name will be included in the release notes for the next version and you'll be listed amongst our other contributors in the next software release paper.

**Pull request merged**